concordia/tests/utils.py

```diff
@@ -104,6 +104,11 @@ def create_asset(
     return asset
 
 
+def clean_up():
+    campaign = Campaign.objects.get(title="Test Campaign")
```

Comment:
Is this actually necessary? Django's test case normally resets the database between runs

Reply:
Well, without the clean up it fails because it tries to run setUp() twice and fails because stuff already exists.

Reply:
That sounds like we might want to use `setUpTestData()` instead: https://docs.djangoproject.com/en/2.1/topics/testing/tools/#django.test.TestCase